orttraining/orttraining/core/optimizer/megatron_transformer.cc

```diff
@@ -39,16 +39,16 @@ const std::initializer_list<ONNX_NAMESPACE::OperatorSetVersion> opset_v9 = {9};
 const std::initializer_list<ONNX_NAMESPACE::OperatorSetVersion> opset_v9_13 = {9, 13};
 const std::initializer_list<ONNX_NAMESPACE::OperatorSetVersion> opset_v12_13 = {12, 13};
 const OpInfo add_info = OpInfo("Add", opset_v7_13_14);
-const OpInfo split_info = OpInfo("Split", opset_v2_11_13, kOnnxDomain, 3);
-const OpInfo reshape_info = OpInfo("Reshape", opset_v5_13);
+const OpInfo split_info = OpInfo("Split", opset_v2_11_13, kOnnxDomain, 3);  // TODO: needs to be updated for opset upgrade
```

Comment:
I dont think this transformer is relevant. @pengwa can you comment.

Reply:
Yes it is needed. It will not be used by ORTModule.